MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -245,7 +245,14 @@ public virtual T Load<T>(string assetName)
             object asset = null;
             if (loadedAssets.TryGetValue(key, out asset))
             {
-                if (asset is T)
+                // Check if we're retrieving a disposed asset
+                // and if so, let's reload it instead
+                bool assetDisposed = false;
+                var disposable = asset as IDisposable;
+                if (disposable != null && disposable.IsDisposed)
```

Comment:
Where does .IsDisposed come from here?

Reply:
Ahah, from a local extension. Got to rework it, hold on.